apollo-compiler/src/main/kotlin/com/apollostack/compiler/SchemaTypeSpecBuilder.kt

```diff
@@ -65,15 +65,15 @@ class SchemaTypeSpecBuilder {
             .build()
 
     /** Returns a generic `Fragments` interface with methods for each of the provided fragments */
-    private fun newFragmentInterfaceSpec(fragments: List<String>): TypeSpec =
+    private fun newFragmentInterfaceSpec(fragments: List<String>, irPkgName: String): TypeSpec =
         TypeSpec.interfaceBuilder(FRAGMENTS_INTERFACE_NAME)
             .addModifiers(Modifier.PUBLIC, Modifier.STATIC)
             .addMethods(fragments.map {
               MethodSpec.methodBuilder(it.decapitalize())
-                  .returns(ClassName.get("", it.capitalize()))
+                  .returns(ClassName.get("$irPkgName.${GraphQLCompiler.FRAGMENT_PACKAGE_PREFIX}", it.capitalize()))
```

Comment:
yeah that would make more sense.
Maybe those builders should be receive a `fragmentPackage` and `typePackage` params instead of one `irPkgName`